CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Fix OpenTelemetry errors when using with Logstasher gem ([#372](https://github.com/alphagov/govuk_app_config/pull/372))
```

Comment:
Ahhh thank you - I was just thinking about this error today and how it would be good to figure out how to get rid of it!